### PR TITLE
Created a simple list of different tools and templates of Neutralinoj…

### PR DESCRIPTION
**Pull request by devanshkansagra**
…s #239

### CODE REVIEW

**Comment by devanshkansagra** [comment]
Please check the pull request.

**Comment by devanshkansagra** [comment]
The list like this I 've created please check. [](https://github.com/neutralinojs/neutralinojs.github.io/assets/125076549/305b8b8d-7cf9-470e-a521-01a52e788bcc
)

**Comment by devanshkansagra** [comment]
I have created list like this
https://github.com/neutralinojs/neutralinojs.github.io/assets/125076549/305b8b8d-7cf9-470e-a521-01a52e788bcc

File: src/pages/styles.module.css [hunk 1]
```diff
@@ -46,3 +50,39 @@
...
+
+@media (max-width: 768px) {
+  .card {
+    flex-direction: column;
+  }
+}
+
+ul li {
+  list-style-type: none;
+}
```
Comment (shalithasuranga) [inline src/pages/styles.module.css:88]: Thanks for sending this pull requestion. A small comment regarding CSS. Could you please avoid adding custom CSS and use existing Docusaurus CSS class names? You can find more about styling from [this link](https://docusaurus.io/docs/styling-layout) and [Infima website](https://infima.dev/). Once you've done these updates, we can merge this pull request. Please let me know if you have questions. Thanks :tada:

**Review by shalithasuranga — CHANGES_REQUESTED** [review]
Please check the inline comment...

File: src/pages/styles.module.css (hunk 1 above)
Comment (devanshkansagra) [reply src/pages/styles.module.css:88]: Sure No Problem

**Comment by devanshkansagra** [comment]
Hey Can I create a new Branch coz my old directory was deleted due to system failure

**Comment by shalithasuranga** [comment]
Hello @devanshkansagra , Sure, no problem :)

**Pull request closed without merging**